src/covid_model_seiir_pipeline/forecasting/model/ode_forecast.py

```diff
@@ -249,7 +249,7 @@ def __init__(self, *args, **kwargs):
 
     def _group_system(self, t: float, y: np.array, infectious: float,
```

Comment:
Is there a docstring for these variables? Looking at this codebase for the first time, I'm confused.

Reply:
What is "screened" in general in this script?